pkg/collector/annotations.go

```diff
@@ -22,9 +22,9 @@ import (
 )
 
 // Annotations return the annotations for OpenTelemetryCollector pod.
-func Annotations(instance v1alpha1.OpenTelemetryCollector) map[string]string {
+func Annotations(instance v1alpha1.OpenTelemetryCollector) (map[string]string, map[string]string) {
```

Comment:
I liked more a separate function  `PodAnnotations` from the original PR. 

Please also update the godocs.